Dockerfile.websat

```diff
@@ -0,0 +1,24 @@
+FROM python:3.8-slim
+
+LABEL vendor=neon.ai \
+    ai.neon.name="neon-iris-websat"
+
+ENV OVOS_CONFIG_BASE_FOLDER neon
+ENV OVOS_CONFIG_FILENAME neon.yaml
+ENV XDG_CONFIG_HOME /config
+
+
+RUN apt update && \
+    apt install -y ffmpeg \
+    && rm -rf /var/lib/apt/lists/*
+
+ADD . /neon_iris
+WORKDIR /neon_iris
+
+RUN pip install wheel && \
+    pip install .[web_sat]
+
+COPY docker_overlay/ /
+EXPOSE 8000
+
+CMD ["iris", "start-websat"]
```

Comment:
It may be easier to maintain one `Dockerfile` with 2 build targets (`gradio` and `websat`?)